@here/harp-mapview-decoder/lib/ThemedTileDecoder.ts

```diff
@@ -35,7 +35,7 @@ export abstract class ThemedTileDecoder implements ITileDecoder {
     }
 
     decodeTile(
-        data: ArrayBufferLike,
+        data: ArrayBufferLike | {},
```

Comment:
I already have a PR for this: https://github.com/heremaps/harp.gl/pull/2198/files and I invited you to review, but I guess you missed it.

Reply:
Oh, sorry, I didn't see that. I just merged it.

Reply:
No worries.